Documentation/ContentElements/Content/Index.rst

```diff
@@ -14,7 +14,7 @@ and "Languages". The "Languages" mode will be discussed later when talking
 about translations.
 
 For now we want to be in "Columns" mode. If that is not the case when
-you move to the *Page* module, please use the menu in the docheader to
+you move to the *Page* module, please use the menu in the doc header to
```

Comment:
The "Getting Started Tutorial" uses the spelling Docheader: https://docs.typo3.org/m/typo3/tutorial-getting-started/10.4/en-us/GeneralPrinciples/GeneralBackendStructure/Index.html

I  would stick to that or change it everywhere.